connector_importer_product/components/product_product/record_handler.py

```diff
@@ -238,10 +247,44 @@ def _find_attr_value(self, attr, attr_column, orig_values):
             attr_value = self.env.ref(sanitize_external_id(orig_val), False)
         if not attr_value and "_value_" not in orig_val:
             # 3rd try w/ auto generated xid
-            value = slugify_one(orig_val).replace("-", "_")
-            xid = f"{attr_column}_value_{value}"
-            attr_value_external_id = sanitize_external_id(xid)
-            attr_value = self.env.ref(attr_value_external_id, False)
+            attr_value_xid = self._make_attribute_value_xid(attr_column, orig_val)
+            attr_value = self.env.ref(attr_value_xid, False)
+        if not attr_value and self.create_attribute_value_if_missing:
+            attr_value = self._create_missing_attribute_value(
```

Comment:
why would `_find_attr_value` create the value?
Shouldn't this be handled by a `_find_or_create_value` method?

Reply:
fine